src/plugins/videoroom-plugin.js

```diff
@@ -865,6 +874,33 @@ class VideoRoomHandle extends Handle {
     throw (error);
   }
 
+  /**
+   * Enable or disable recording for all participants in a room, while the conference is in progress.
+   *
+   * @param {object} params
+   * @param {number|string} params.room - The room where the kick is being requested
+   * @param {string} params.secret - The optional secret for the operation
+   * @param {boolean} params.record - Whether participants in this room should be automatically recorded or not
```

Comment:
You re-used the same comment from janus docs, but I feel that statement is not accurate.
This param will enable or disable recording for all participants also while running the conference.